public/doc/swagger-2.yaml

```diff
@@ -680,6 +680,13 @@ definitions:
         type: string
         title: ServiceOfferingRef
         example: The Service Offering Reference
+  AddPortfolioItem:
+    type: object
+    properties:
+      portfolio_item_id:
+        type: string
+        title: PortfolioItemId
```

Comment:
@syncrou 
Should this have spaces Portfolio Item Id?

Reply:
I can make that change.  I'll change `ServiceOfferingRef` on line 681 as well.